dist/table_of_contents.mustache

```diff
@@ -0,0 +1,106 @@
+              <div>
+                  <div class="version-header">
+                      <h2>Documentation for the various versions of Postman Schemas</h2>
+                  </div>
+                  <div>
+                      <h3>Collections</h3>
+                      <table class="table table-striped table-bordered table-hover">
+                          <thead>
+                              <tr>
+                                  <th>Version</th>
+                                  <th>Status</th>
+                                  <th>Documentation</th>
+                                  <th>Raw Schema</th>
+                              </tr>
+                          </thead>
+                          <tbody>
+                              {{#versions}}
+                              <tr>
+                                  <td>{{raw}}</td>
+                                  <td><span class="label label-{{bootstrapClass}}">{{status}}</span></td>
+                                  <td><a href="./json/collection/{{raw}}/docs/index.html" target="_blank">View
+                                          Documentation</a></td>
+                                  <td><a href="./json/collection/{{raw}}/collection.json" target="_blank">JSON Schema</a></td>
+                              </tr>
+                              {{/versions}}
+                          </tbody>
+                      </table>
+                  </div>
+                  <div>
+                      <h3>Legend</h3>
+                      <table class="table table-striped table-bordered table-hover">
+                          <tbody>
+                              <tr>
+                                  <td><span class="label label-warning">draft</span></td>
+                                  <td>Draft - unstable and liable to be changed anytime</td>
+                              </tr>
+                              <tr>
+                                  <td><span class="label label-primary">rc</span></td>
+                                  <td>Release Candidate - Only minor changes possible in the future</td>
+                              </tr>
+                              <tr>
+                                  <td><span class="label label-success">stable</span></td>
+                                  <td>Stable - No changes likely. In exceptional cases, changes may occur, but they will
+                                      be backwards compatible</td>
+                              </tr>
+                          </tbody>
+                      </table>
+                  </div>
+              </div>
+          </div>
+      </div>
+      <div class="footer-container open-sans">
+        <span>&copy;
+          <script type="text/JavaScript">document.write(new Date().getFullYear());</script>
+          Postdot Technologies, Inc.
+        </span>
+        <span>All Rights Reserved</span>
+      </div>
+      <script type="text/javascript">
```

Comment:
This should be in the head: https://developers.google.com/tag-manager/quickstart